demos/flight-booking/src/index.tsx

```diff
@@ -5,13 +5,15 @@ import App from './App';
 import { SpeechProvider } from "@speechly/react-client";
 import * as serviceWorker from './serviceWorker';
 import { LogKit } from "@speechly/logkit"
+import { DemoNavigation } from '@speechly/demo-navigation';
 
 const appId = '1ea63538-f95c-4259-b8af-923994424137'
 
 ReactDOM.render(
   <React.StrictMode>
     <SpeechProvider appId={appId}>
       <LogKit appName="flight-booking" appVersion={100} >
+        <DemoNavigation />
```

Comment:
This demo is embedded on this page: https://www.speechly.com

We probably don't want the navigation bar to appear there?